pkgs/applications/networking/cluster/terraform-providers/providers.json

```diff
@@ -804,15 +768,6 @@
     "spdx": "MPL-2.0",
     "vendorHash": "sha256-6ePPxdULuTzLdVzzr12BjLu/lBN+5yIUq8U8FVUw/PM="
   },
-  "nsxt": {
-    "hash": "sha256-TOoRtCKdR1fBjk39dbMgBd7pDJGfjvkQAqfpJzWRwRg=",
```

Comment:
This one also had a license but probably not picked up because it came without a .md extension: https://github.com/vmware/terraform-provider-nsxt/blob/master/LICENSE.txt